server/plugin/command.go

```diff
@@ -315,29 +307,105 @@ func createMDForLogsList(results []string, fallback string) string {
 	return res
 }
 
-func (c *command) authUser(_ ...string) (string, error) {
-	return fmt.Sprintf("Server : %s\nUser : %s", c.splunk.User().Server, c.splunk.User().UserName), nil
+func addSubCommands(splunk *model.AutocompleteData) {
+	splunk.AddCommand(createAlertCommand())
+	splunk.AddCommand(createAuthCommand())
+	splunk.AddCommand(createlogCommand())
```

Comment:
```suggestion
	splunk.AddCommand(createLogCommand())
```

Reply:
Fixed in [`d9daacd` (#107)](https://github.com/mattermost/mattermost-plugin-splunk/pull/107/commits/d9daacd2957789d8f8a3e5de73f1da52d8aa4b43)